### ISSUE TITLE
Leadership Team Page Update

### ISSUE THREAD

**Issue by DairyProducts** [theunitedeffort/theunitedeffort.org#633]
https://www.theunitedeffort.org/about/team/

The length and style of each about card differs rather greatly, resulting in a lot of whitespace on desktop. Some cards are very short and some are much longer than others. The first few cards have also never been updated since the time when theunitedeffort.org was a Razor Pages site (although I guess there isn't really a need to update these often). Perhaps we could ask everyone on the leadership panel to write or update their about card to be within a certain range of characters.

Alternatively, to solve the whitespace issue, we could organize the cards such that vertically each card is the same distance away from the one on top of it.

**Comment by trevorshannon** [comment]
That's a good idea. Perhaps we can do a review of the bios, though it was hard enough to get bios from everyone in the first place!

I think the alternate solution of aligning cards may decrease readability, though I guess I'd have to see it to be sure.

**Comment by DairyProducts** [comment]
I'm looking at this one a year and a half later and it looks like the problem is mostly resolved for now on the About page (maybe some board members moved around). The team link in the original issue also no longer exists (it leads to a 404).

**Comment by trevorshannon** [comment]
Yes, we did shorten the bios at some point when we moved the content from its own dedicated page to its new location [here](https://www.theunitedeffort.org/about/)
